src/Arcus.Observability.Tests.Unit/Arcus.Observability.Tests.Unit.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
     <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="Serilog.AspNetCore" Version="3.2.0" />
```

Comment:
Added this because we could use the `SerilogLoggerFactory` that creates a Microsoft `ILogger` from a Serilog `ILogger` so we can use our `LogMetric`, `LogRequest`, ... extensions.